### PR TITLE
feat(3208): Stats of the whole Basilisk marketplace

### LINKED ISSUE #3208: Stats of the whole Basilisk marketplace
Show statistics of total
- buys
- offers (count, total value)
- listed items for sale
- total items minted
- total collections 
- total amount of wallet addresses 

Anything else?

### LINKED ISSUE #3255: Expand stats of whole Basilisk Marketplace
> I'm guessing this means offers with status ACCEPTED and not WITHDRAWN/EXPIRED/ACTIVE. Maybe add column for count/total of ACTIVE?

Make sense if we can expand it to 

- amount of 
  - expired offers and total value 
  - withdrawn offers and total value 
  - active offers and total value

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/3208#issuecomment-1164498397_

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3208, closes #3255
- [x] Requires deployment of new snekk @vikiival 
- [x] New screen for stats about BSX marketplace 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/bsx/Offer/StatsOverview.vue [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  <div>
+    <div class="level my-4 collection is-align-items-center mb-5">
+      <div class="level-item has-text-centered">
+        <div>
+          <p class="title">{{ buys }}</p>
+          <p class="heading">
+            {{ $t('statsOverview.buys') }}
+          </p>
+        </div>
+      </div>
```
Comment (vikiival) [inline components/bsx/Offer/StatsOverview.vue:11]: Make component. 

DRY

**Review by vikiival — CHANGES_REQUESTED** [review]

**Comment by vikiival** [comment]
What i would do is 


```ts
export function countOf(value: SomethingWithCount) {
  return value.count || value.totalCount
}
```

then in vue you can use

```ts
this.buys = countOf(buys)
```

**Comment by yangwao** [comment]
Let's add one more column with relative time 

image

> Link to NFT
Let's rename it to "Item"

**Comment by petersopko** [comment]
> Let's add one more column with relative time, like we have in history

added relative time, not the same way as history, since history relative time is currently broken on BSX
- #3280 

> Let's rename it to "Item"\

done

**Review by vikiival — APPROVED** [review]

**Pull request merged**